integration/src/main/java/scripts/testscripts/ReferencedTdrSnapshotLifecycle.java

```diff
@@ -0,0 +1,185 @@
+package scripts.testscripts;
```

Comment:
Elsewhere we call this DataRepoSnapshot.

Reply:
sgtm, renamed the test to `ReferencedDataRepoSnapshotLifecycle`